packages/friends/yml/es/john-burnyeat.yml

```diff
@@ -7,7 +7,10 @@ born: 1631
 died: 1690
 quotes:
   - source: George Fox
-    text: TODO
+    text:
+      '[John Burnyeat] era un amigo y hermano fiel, y un ministro competente de
+      Jesucristo…un anciano, y una columna en la casa de Dios...un verdadero apóstol de
```

Comment:
Hmmm... looks like one of these is a single-char ellipses, and one is three dots.  We should maybe look at the deploy preview and see which one looks better and make them both the same, I think.